openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -127,10 +127,6 @@ def get_seconds_since_epoch(date_time):
             ('clean_id', ('#',)),
             ('has_ended', ()),
             ('has_started', ()),
-            ('start_datetime_text', ('SHORT_DATE',)),
-            ('start_datetime_text', ('DATE_TIME',)),
-            ('end_datetime_text', ('SHORT_DATE',)),
-            ('end_datetime_text', ('DATE_TIME',)),
             ('may_certify', ()),
```

Comment:
Was throwing errors via the 'CourseDescriptorWithMixins' object. 

[Build 29200](https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/29200/). This appears to be linked to the CourseDescriptor module from xmodule.course_module